src/Microsoft.CodeQuality.Analyzers/Core/QualityGuidelines/AbstractRemoveEmptyFinalizers.cs

```diff
@@ -55,8 +55,8 @@ public override void Initialize(AnalysisContext analysisContext)
             });
         }
 
-        protected static bool InvocationIsConditional(IMethodSymbol methodSymbol, INamedTypeSymbol conditionalAttributeSymbol) =>
-            methodSymbol.GetAttributes().Any(n => n.AttributeClass.Equals(conditionalAttributeSymbol));
+        protected static bool InvocationIsConditional(IMethodSymbol methodSymbol, INamedTypeSymbol? conditionalAttributeSymbol) =>
+            conditionalAttributeSymbol != null && methodSymbol.GetAttributes().Any(n => n.AttributeClass.Equals(conditionalAttributeSymbol));
```

Comment:
💭 I thought I saw a `HasAttribute` method for this?

```suggestion
            HasAttribute(methodSymbol, conditionalAttributeSymbol);
```